### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by frauniki**
- Update Go and all dependencies to the latest version

### CODE REVIEW

**Comment by frauniki** [comment]
@FeLvi-zzz 
Could you please configure renovate?

**Comment by FeLvi-zzz** [comment]
@frauniki 
I've done, does #55 include #56? if so, I'll close #56.
https://github.com/FeLvi-zzz/tentez/pull/56

File: ui.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//nolint:errcheck
```
Comment (FeLvi-zzz) [inline ui.go:1]: note
`fmt.Fscan(c.in, &input)` is not handling err

**Review by FeLvi-zzz — APPROVED** [review]

**Comment by frauniki** [comment]
> @frauniki I've done, does #55 include #56? if so, I'll close #56.

Yes, it includes.

**Pull request merged**